themes/api.jquery.com/index.php

```diff
@@ -1,4 +1,13 @@
-<?php get_header(); ?>
+<?php
+	// Set banner in post meta for the header to display it
+	update_post_meta(
```

Comment:
This appears to perform a database modification on GET requests. I don't think we can do that at scale. If it's unconditional like this, I suppose we can move this as default or instead of where the banner meta is consumed, for api pages.

Reply:
I wasn't aware it made it all the way to the database. Perhaps a hook instead?

Reply:
Can we instead define a global banner map from banner ID to its contents & the list of sites where it should be displayed?

Reply:
Can we instead define a global banner map from banner ID to its contents & the list of sites where it should be displayed?

Reply:
Can we instead define a global banner map from banner ID to its contents & the list of sites where it should be displayed?

Reply:
going to look into setting this via puppet